octoprint_mrbeam/static/js/material_settings.js

```diff
@@ -32,46 +46,30 @@ $(function () {
                 img: 'anodized_aluminum.jpg',
                 description: gettext("Dark anodized aluminum can be engraved. Works on iPhones."),
                 hints: gettext("Requires very precise focus. Anodized aluminum turns brighter through laser engraving. Therefore we suggest to invert photos for engravings."),
-                params: {
-                    'MrBeamII-1.0': {
-                        '000000': {
-                            engrave: {eng_i: [0, 100], eng_f: [1000, 30], eng_pierce: 0, dithering: false},
+                colors: {
```

Comment:
remove all the colors values but keep one as a comment for future developer's reference